terraform/deployments/govuk-publishing-platform/app_publisher.tf

```
@@ -1,3 +1,67 @@
+locals {
+  publisher_app_name = "publisher"
+
+  publisher_defaults = {
+    cpu    = 512  # TODO parameterize this
+    memory = 1024 # TODO parameterize this
+
+    environment_variables = merge(
+      local.defaults.environment_variables,
+      {
+        ASSET_HOST                       = local.defaults.asset_host
+        BASIC_AUTH_USERNAME              = "gds"
+        EMAIL_GROUP_BUSINESS             = "test-address@digital.cabinet-office.gov.uk"
+        EMAIL_GROUP_CITIZEN              = "test-address@digital.cabinet-office.gov.uk"
+        EMAIL_GROUP_DEV                  = "test-address@digital.cabinet-office.gov.uk"
+        EMAIL_GROUP_FORCE_PUBLISH_ALERTS = "test-address@digital.cabinet-office.gov.uk"
+        FACT_CHECK_SUBJECT_PREFIX        = "dev"
+        FACT_CHECK_USERNAME              = "govuk-fact-check-test@digital.cabinet-office.gov.uk"
+        GOVUK_APP_DOMAIN_EXTERNAL        = var.external_app_domain
+        GOVUK_APP_NAME                   = local.publisher_app_name
+        GOVUK_APP_ROOT                   = "/app"
+        # TODO: how does GOVUK_ASSET_ROOT relate to ASSET_HOST? Is one a function of the other? Are they both really in use? Is GOVUK_ASSET_ROOT always just "https://${ASSET_HOST}"?
+        GOVUK_ASSET_ROOT    = "https://assets.test.publishing.service.gov.uk"
+        GOVUK_STATSD_PREFIX = "govuk-ecs.app.${local.publisher_app_name}"
+        # TODO: looks suspicious
+        PLEK_SERVICE_CONTENT_STORE_URI  = "https://www.gov.uk/api"
```

Maybe we could take this opportunity to fix this one?

```suggestion
        PLEK_SERVICE_CONTENT_STORE_URI  = local.defaults.content_store_uri
```

(assuming that's actually set in the defaults, which it probably isn't)